srcpkgs/cryfs/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'cryfs'
+pkgname=cryfs
+version=0.10.1
+revision=1
+create_wrksrc=yes
+build_style=cmake
+configure_args="-DCRYFS_UPDATE_CHECKS=off
+ -DOpenMP_CXX_FLAGS=-I${XBPS_CROSS_BASE}/usr/include
+ -DOpenMP_CXX_LIB_NAMES=omp
+ -DOpenMP_omp_LIBRARY=${XBPS_CROSS_BASE}/usr/lib/libomp.so"
+hostmakedepends="git make python"
+makedepends="boost-devel fuse-devel libcurl-devel libomp-devel libressl-devel"
+depends="fuse"
+short_desc="Cryptographic filesystem for the cloud"
+maintainer="Andy Weidenbaum <archbaum@gmail.com>"
+license="LGPL-3.0-only"
+homepage="https://www.cryfs.org"
+changelog="https://github.com/cryfs/cryfs/raw/master/ChangeLog.txt"
+distfiles="https://github.com/cryfs/cryfs/releases/download/${version}/${pkgname}-${version}.tar.gz"
+checksum=a01e15dd4520a7bb3a040d502b09e43e755f12b35f9eff88dc4d00b6286fbb1a
+
+if [ "${XBPS_CHECK_PKGS}" ]; then
+	configure_args+=" -DBUILD_TESTING=on"
```

Comment:
Is the time spent building test binaries and stuff enough to justify this ?

Reply:
building test support takes significantly more memory and time in my limited vbox testing

Reply:
cc @smessmer, is there any point to building test support by default?

Reply:
Musl and voidlinux are not part of the CryFS CI and as shown by the issues in 0.10.0, things seem prone to be incompatible. I recommend running the tests whenever a new release package is built to make sure changes in future CryFS versions don't break anything. They probably don't need to be run on the users PC at installation time. While that would be even safer, it would also be a significant installation time cost.

Reply:
seems good to me, people updating the package can run the tests.